requirements.txt

```diff
@@ -2,4 +2,6 @@ pyqt5==5.15.2
 qdarkstyle==2.8
 nidaqmx==0.5.7
 black
-pre-commit
\ No newline at end of file
+pre-commit
+serial~=0.0.97
+numpy~=1.20.2
```

Comment:
we like having fixed versions better usually.  We can change these two line to use `==`. Also I believe Bin said he is using `pyserial` package. We should be installing that.  Maybe `pyserial==3.5`?